Tests/test_CombinatorialPromoter.py

```diff
@@ -0,0 +1,153 @@
+
+# this file made by Andrey Shur, 5/5/2020
```

Comment:
Hi Andrey, 
Would you mind renaming this file to test_combinatorial_promoter.py, so it'd follow the naming pattern of the other test files? Many Thanks!